defaults/config/install_profile/the_build.install

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * @file
+ * Install, update and uninstall functions for the installation profile.
+ */
+
+use Drupal\user\Entity\User;
+use Drupal\user\RoleInterface;
+use Drupal\shortcut\Entity\Shortcut;
+
+/**
+ * Implements hook_install().
+ *
+ * Perform actions to set up the site for this profile.
+ *
+ * @see system_install()
+ */
+function the_build_install() {
+  // Set front page to "node".
+  \Drupal::configFactory()->getEditable('system.site')->set('page.front', '/node')->save(TRUE);
+
+  // Allow visitor account creation with administrative approval.
+  $user_settings = \Drupal::configFactory()->getEditable('user.settings');
+  $user_settings->set('register', USER_REGISTER_ADMINISTRATORS_ONLY)->save(TRUE);
+
+  // Assign user 1 the "administrator" role.
+  $user = User::load(1);
+  $user->roles[] = 'administrator';
+  $user->save();
+
+  // Allow authenticated users to use shortcuts.
+  user_role_grant_permissions(RoleInterface::AUTHENTICATED_ID, ['access shortcuts']);
+
+  // Populate the default shortcut set.
+  $shortcut = Shortcut::create([
+    'shortcut_set' => 'default',
+    'title' => t('Manage Taxonomy'),
+    'weight' => -20,
+    'link' => ['uri' => 'internal:/admin/structure/taxonomy'],
+  ]);
+  $shortcut->save();
+
+  $shortcut = Shortcut::create([
+    'shortcut_set' => 'default',
+    'title' => t('Manage Menus'),
+    'weight' => -19,
+    'link' => ['uri' => 'internal:/admin/structure/menu'],
+  ]);
+  $shortcut->save();
+
+  $shortcut = Shortcut::create([
+    'shortcut_set' => 'default',
+    'title' => t('Status Report'),
+    'weight' => -18,
+    'link' => ['uri' => 'internal:/admin/reports/status'],
+  ]);
+  $shortcut->save();
+
+  // Enable the admin theme.
+  \Drupal::configFactory()->getEditable('node.settings')->set('use_admin_theme', TRUE)->save(TRUE);
+
+  // Configure time and date settings.
+  $date_settings = \Drupal::configFactory()->getEditable('system.date');
+  $date_settings->set('country.default', 'US');
+  $date_settings->set('timezone.default', 'America/Chicago');
+  $date_settings->set('timezone.user.configurable', FALSE);
+  $date_settings->save(TRUE);
+}
```

Comment:
If we remove the config, it might be good to add/remove modules with some code like this: https://github.com/palantirnet/akamai/blob/e267bbf5ebe8ea6ff59b07caecc1e9dd93c53fb4/web/modules/custom/ak_utilitiy/ak_utility.install#L11-L20

```php
  $old = ['automated_cron', 'big_pipe', 'contact', 'history', 'search', 'tour'];
  $new = ['admin_toolbar', 'admin_toolbar_tools', 'config_split', 'workbench_tabs'];

  \Drupal::service('module_installer')->install($new);
  \Drupal::service('module_installer')->uninstall($old);
```

I left out devel and stage_file_proxy since they'll have to be set up split to the development environment.

Reply:
I don't think that we can extend the 'standard' install profile, given that [#1356276 Allow profiles to define a base/parent profile and load them in the correct order](https://www.drupal.org/project/drupal/issues/1356276) is still open, which means that we wouldn't be uninstalling those modules like this... unless we provided the code that is currently an install profile as a module instead -- is that what you're thinking here?

The challenge I found with the standard install profile is that it is difficult to uninstall some of the modules that are enabled by default -- specifically, `contact` and `comment`, because the entity types/fields have to be explicitly deleted before they can be uninstalled. I think it's important to uninstall those modules because they present forms for user input that we don't usually use -- and that's not great, if we forget to uninstall them later.

... but if we don't use the standard install profile, we don't get things like the default block placement, the 'basic' block type, and the article/page content types... etc., etc.

So if I change this to be a utility module, then I would probably just do the extra work to fully uninstall `contact` and `comment`.

Reply:
Yea, I didn't fully think it out before, but without having any config then moving this into a module (like `the_build_utility` or `the_build_installer` as you suggested via Slack) I think would be the right path. 

You present some good points, and I'm torn on where to start. I often think Minimal might be a better starting point since we're always overriding something in Standard, but then I look and Minimal is soooo little. I think your idea above sounds like the best compromise. With the existing install code it shouldn't (🤞) be too much work to remove those fields and disable the modules. That way instead of trying to track the changes to Standard with new Core updates we just have to make any modifications to our install steps, which should be much easier to maintain.